apps/daimo-mobile/src/view/shared/HistoryList.tsx

```diff
@@ -156,25 +159,39 @@ function HeaderRow({ title }: { title: string }) {
   );
 }
 
+function getFromTo(op: DisplayOpEvent): [Address, Address] {
+  if (op.type === "transfer") {
+    return [getAddress(op.from), getAddress(op.to)];
+  } else {
+    if (op.noteStatus.claimer?.addr === op.noteStatus.sender.addr) {
+      // Self-transfer via payment link shows up as two payment link transfers
+      return [getAddress(op.from), getAddress(op.to)];
```

Comment:
why do we have to call `getAddress` on op.from but not on op.noteStatus.sender.addr

both should already be Address i think

Reply:
it used to be getAddress before so I was keeping it, but yeah I don't see a reason for it to be now, removed all getAddress 👍